package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lab-steven",
+  "version": "1.0.0",
+  "description": "lab-01 401JS",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/StevenJohnson86/lab-01-modular-patterns.git"
+  },
+  "keywords": [
+    "nodejs",
+    "test",
+    "mocha",
+    "require"
+  ],
+  "author": "CF StevenJohnson",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/StevenJohnson86/lab-01-modular-patterns/issues"
+  },
+  "homepage": "https://github.com/StevenJohnson86/lab-01-modular-patterns#readme",
+  "devDependencies": {
+    "argv": "0.0.2",
```

Comment:
argv is built into node, so you don't need to include it as a dependency :)   I've seen similar package.json files on stack-overflow, but it might've been for older versions of node. it does show you're using your resources, though!